src/tiled/newtemplatedialog.ui

```diff
@@ -59,53 +59,25 @@
      <property name="title">
       <string>Group</string>
      </property>
-     <layout class="QFormLayout" name="formLayout">
-      <item row="0" column="0">
-       <widget class="QRadioButton" name="ExistingGroupRadioButton">
+     <layout class="QGridLayout" name="gridLayout_2">
+      <item row="2" column="0">
+       <widget class="QLabel" name="groupLable">
         <property name="text">
-         <string>Save to an existing group</string>
-        </property>
-        <property name="checked">
-         <bool>true</bool>
+         <string>Group:</string>
         </property>
        </widget>
       </item>
-      <item row="0" column="1">
-       <widget class="QComboBox" name="groupsComboBox">
-        <property name="modelColumn">
-         <number>0</number>
-        </property>
-       </widget>
-      </item>
-      <item row="1" column="0">
-       <widget class="QRadioButton" name="newGroupRadioButton">
-        <property name="enabled">
-         <bool>true</bool>
-        </property>
+      <item row="2" column="3">
+       <widget class="QPushButton" name="createGroupButton">
         <property name="text">
-         <string>Save to a new group</string>
-        </property>
-        <property name="checkable">
-         <bool>true</bool>
-        </property>
-        <property name="checked">
-         <bool>false</bool>
+         <string>Create new Group</string>
```

Comment:
I think this button should probably use the same "new file" icon as used in the templates dock. This could be its tool tip.